lib/util/merge-confidence/index.ts

```diff
@@ -25,14 +25,51 @@ export const confidenceLevels: Record<MergeConfidence, number> = {
 export function initConfig(): void {
   apiBaseUrl = getApiBaseUrl();
   token = getApiToken();
+  supportedDatasources = parseSupportedDatasourceString() ?? [
+    ...presetSupportedDatasources,
+  ];
+
   if (!is.nullOrUndefined(token)) {
     logger.debug(`Merge confidence token found for ${apiBaseUrl}`);
   }
 }
 
+export function parseSupportedDatasourceString(
```

Comment:
I think we can also remove this unused parameter. we never pass it besides tests.